common/compat.c

```diff
@@ -1051,3 +1051,13 @@ p11_ascii_toupper (int c)
 		return 'A' + (c - 'a');
 	return c;
 }
+
+int
+p11_strcasecmp (const char *s1,
```

Comment:
I will add "ascii" to the name, but I am not going to reduce the functionality of the function by making it return bool when its supposed to be a general string comparison function. Otherwise whats the point of having it in compat.h when its supposed to be a single use only? In such case I could have left it local to the generate-keypair.c file.

Reply:
There is no point in making the internal function as generic as possible, either. If we make it a general string comparison function, it needs to work correctly (e.g., it works based on the ordering axioms as checked by [sortcheck](https://github.com/yugr/sortcheck)); it can be overly complicated and out of scope of our use-case.

Limiting the responsibility of the function is also beneficial to avoid programming errors (e.g., https://github.com/libreswan/libreswan/blob/a4bed3681646f19bc3ee2783f88a3257e92e0659/docs/ProgrammingConventions.txt#L65).